src/RazorSdk/Tasks/StaticWebAssets/GenerateStaticWebAssetsDevelopmentManifest.cs

```diff
@@ -64,8 +68,8 @@ public override bool Execute()
 
             var discoveryPatternsByBasePath = DiscoveryPatterns
                 .Select(ComputeDiscoveryPattern)
-                .GroupBy(p => p.BasePath,
-                 (key, values) => (key.Split('/'), values));
+                .GroupBy(p => p.BasePath == p.BasePath ? "" : p.BasePath,
```

Comment:
This looks wrong. did you mean `p.BasePath == BasePath`?

Reply:
```suggestion
                .GroupBy(p => p.BasePath == p.BasePath ? string.Empty : p.BasePath,
```

Reply:
Yes, I noticed too and fixed it.

The goal of this is to make sure that we don't apply the BasePath to patterns in the current project since the (base path for the project will match that from the pattern)

That said, I'll change this after preview7 to include the Source (essentially the PackageId) and do it based on that, so that we make sure we only do it for the current project.